### DISCUSSION TITLE
Broken build due to older ldd version

### DISCUSSION THREAD

**Discussion by GhostMaster69-dev** [ROM-builders/temporary#16174] [Ideas]
getting error: version `GLIBC_2.36' not found
update your docker image to ubuntu 22.10 for newer ldd version with glibc 36 (where Current glibc version is 35) required by custom clangs built on Ubuntu 22.10
You can check yourself by running following command:
$ ldd --version

**Comment by Apon77** [comment]
22.10 is not LTS, so it is not good to update to that version. You better stop using that clang or something else. It's not an issue from our side. ROMs can be even compiled on ubuntu 14.04. so, please skip this topic at least.

**Comment by GhostMaster69-dev** [comment]
> 22.10 is not LTS, so it is not good to update to that version. You better stop using that clang or something else. It's not an issue from our side. ROMs can be even compiled on ubuntu 14.04. so, please skip this topic at least.

Yeah fine
